app/src/main/report/actionCreators/reportPageActionCreators.ts

```diff
@@ -14,9 +14,14 @@ export const reportPageActionCreators = {
                 type: ReportTypeKeys.REPORT_VERSION_DETAILS_FETCHED,
                 data: null
             } as ReportVersionDetailssFetched);
+            dispatch({
```

Comment:
I'm bit confused by this - why set the changelog to null here and then immediately reset it to data fetched by the API on line 24?

Reply:
Ah ok, I see you're copying the logic with the version details. I'm not 100% sure why that's being done or whether it's necessary, but my guess is that it's so when you change from one report page to another, the previous report's content doesn't linger. This wouldn't be an issue with the changelog if you fetched it here. 

However see comment below about where I think the changelog should be fetched - in that case, this approach of first setting it to null might well be needed, to avoid stale content being shown when the component first mounts.